### PR TITLE
solution

### PR DESCRIPTION
**Pull request by Andrey1104**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -1,2 +1,64 @@
...
+            self.size -= 1
+
+    def resize(self) -> None:
+        temp_table = self.table
+        self.capacity *= 2
+        self.table = [None] * self.capacity
+        self.size = 0
+        for element in temp_table:
+            if element:
+                self.__setitem__(element[1], element[2])
```
Comment (vsmutok) [inline app/main.py:50]:
```suggestion
                self[element[1]] = element[2]
```

**Review by vsmutok — CHANGES_REQUESTED** [review]

File: app/main.py (hunk 1 above)
Comment (Andrey1104) [reply app/main.py:50]: can you comment in more detail?

Comment (Andrey1104) [reply app/main.py]: can you comment in more detail?

Comment (vsmutok) [reply app/main.py]: `self[element[1]] = element[2]` is a more straightforward and concise way to set a value for a specific key. It directly accesses the dictionary element and is self-explanatory.

**Review by vsmutok — APPROVED** [review]
Good Job 👍

File: app/main.py (hunk 1 above)
Comment (Andrey1104) [reply app/main.py]: but i can set value with setter, is it a mistake to not accept the request?

**Comment by Andrey1104** [comment]
I try to use different methods so as not to forget them

**Comment by Andrey1104** [comment]
соу мени нью інформейшн ін май хед, со ші експлоуд вері суун)))))

**Pull request closed without merging**